### ISSUE TITLE
Having compatibility issue with Next.js

### ISSUE THREAD

**Issue by rockchalkwushock** [cloudinary/cloudinary-react#8]
I have opened this issue with the Next.js dev team as well.

You can find the issue [here](https://github.com/zeit/next.js/issues/1061).

I have created a simple example in a throw away repo to try and figure out the problem I am encountering and it can be found [here](https://github.com/rockchalkwushock/test-repo). The _README.md_ has a synopsis of what is happening as well as links to issues & code.

Basically the gist is upon using `<Image />` in my Next.js project I am met with an error message & warning from Next.js. I am working on looking at the source code for both projects to see what could be causing this problem on my own but definitely am reading Arabic in some cases. Any assistance that can be provided would be greatly appreciated.

**Comment by roeeba** [comment]
Hi @rockchalkwushock  I understand from the correspondence on [this thread](https://github.com/zeit/next.js/issues/1061) that this is an internal Next.js issue. Can please let us know if you still need our assistance in this matter?

**Comment by rockchalkwushock** [comment]
@roeeba 

Sorry I had meant to go ahead and close this as the issue pertains to the Next.js source code.
